docs/especificacao.md

```diff
@@ -192,36 +192,39 @@ d)  A conta do patrocinador é excluída do sistema. <br>
 
 **Pós-condições**: A conta do patrocinador deve existir ou ser excluída, conforme as ações realizadas.
 
-#### Gerenciar perfil das ONGs (CSU04) LEONARDO
+#### Gerenciar perfil das ONGs (CSU04)
 
-**Sumário**: Um breve resumo do que o caso de uso faz e qual o seu objetivo principal.
+**Sumário**: Este caso de uso permite que o representante da ONG gerencie as informações do seu perfil no sistema. O objetivo principal é garantir que o representante da ONG possa atualizar seus dados de contato, criar demandas para recrutar voluntários e acompanhar suas demandas e possíveis voluntários.
 
-**Ator Primário**: Adicione o ator principal.
+**Ator Primário**: ONG.
 
-**Ator Secundário**: Adicione o ator secundário.
+**Ator Secundário**: Administrador.
 
-**Pré-condições**: Condições que devem ser atendidas antes que o caso de uso possa ser executado.
+**Pré-condições**: A ONG deve estar cadastrada no sistema com um login e senha válidos.
 
-**Fluxo Principal**:
+**Fluxo Principal**: Criar perfil da ONG.
 
-1)  Coloque aqui o fluxo.
-2)  Coloque aqui o fluxo.
-3)  Coloque aqui o fluxo.
-4)  Coloque aqui o fluxo...
+1)  A ONG acessa a página de cadastro do sistema.
+2)  Insere seus dados.
+3)  O sistema valida os dados fornecidos.
+4)  O perfil da ONG é criado.
 
-**Fluxo Alternativo**: Coloque aqui o nome fluxo alternativo caso houver.
+**Fluxo Alternativo**: Editar perfil da ONG
 
-a)  Descreva o fluxo alternativo caso houver. <br>
-b)  Descreva o fluxo alternativo caso houver. <br>
-c)  Descreva o fluxo alternativo caso houver... <br>
+a) A ONG faz login no sistema e acessa a área de perfil. <br/> 
```

Comment:
Aqui também seria representante da ONG. Os atores são: representante da ONG, voluntário de TI, patrocinador e administrador.

Então tudo onde for função do ator seriam com esses nomes.